queries.sql

```diff
@@ -142,4 +142,48 @@ GROUP BY species;
 
 SELECT AVG(escape_attempts)
 FROM animals
-WHERE date_of_birth BETWEEN '1990-01-01' AND '2000-12-31';
\ No newline at end of file
+WHERE date_of_birth BETWEEN '1990-01-01' AND '2000-12-31';
+
+
+SELECT name
+FROM animals
+JOIN owners ON animals.owners_id = owners.id
+WHERE owners.full_name = 'Melody Pond';
+
+
+SELECT animals.name
+FROM animals
+INNER JOIN species ON animals.species_id = species.id
+WHERE species.name = 'Pokemon';
+
+
+SELECT full_name,
+       name
+FROM owners
+LEFT JOIN animals ON owners.id = animals.owners_id;
+
+
+SELECT COUNT(*)
+FROM animals
+GROUP BY (species_id);
+
```

Comment:
- [ ] Its great that you are counting the animals per species but you didn't mention the names of the types of animals you are counting. You can make your query like this :+1:

```
SELECT species.name, COUNT(*) FROM animals
LEFT JOIN species
ON animals.species_id = species.id
GROUP BY species.name
```

Reply:
@HAMMAS-SALEEM much appreciated. Thanks for the review💗😁